subprojects/ci/clear-k8s-deployments/build.gradle.kts

```diff
@@ -0,0 +1,40 @@
+import com.avito.kotlin.dsl.getMandatoryStringProperty
+import com.avito.kotlin.dsl.getOptionalStringProperty
+
+//import com.avito.utils.gradle.buildEnvironment
+
+buildscript {
+    val infraVersion: String by project
+    dependencies {
+        classpath("${Dependencies.gradle.avito.kotlinDslSupport}:$infraVersion")
+//        classpath("${Dependencies.gradle.avito.buildEnvironment}:$infraVersion")
+    }
+}
+plugins {
+    id("kotlin")
+}
+
+dependencies {
+    api(Dependencies.kubernetesClient)
+    api(project(":subprojects:common:teamcity-common"))
+    api(Dependencies.kotlinXCli)
+}
+
+// todo add if ci
+//if(project.buildEnvironment is com.avito.utils.gradle.BuildEnvironment.CI) {
+if (project.getOptionalStringProperty("ci", "false").toBoolean()) {
+    tasks.register("run", JavaExec::class.java) {
+        main = "com.avito.ci.ClearK8SDeploymentsMain"
```

Comment:
Why don't we wrap this logic into gradle plugin?

In CLI apps we have to solve again common issues.

Reply:
How do you see the plugin solves these problems?

Reply:
What I name as issues:
- Logging, monitoring
- Integration tests
- ... similar stuff

I know that it is easy in simple CLI libraries. But we use gradle plugins to deal with them in a common manner.
I'm not against CLI libraries. I just want to be sure that we can manage these issues easily and have appropriate utils for it.

Reply:
Example: 
in ClearDeployments we could reuse TeamcityCredentials, KubernetesCredentials and do no pass these parameters in another way.

Reply:
I understand but a plugin is something for projects.
This is no plugin case. But I agree we need possibilities to reuse our logic